packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanBar.tsx

```diff
@@ -59,20 +52,19 @@ function toPercent(value: number) {
   return `${(value * 100).toFixed(1)}%`;
 }
 
-function SpanBar(props: TInnerProps) {
```

Comment:
what was the purpose of InnerProps before? Can we have everything merged into one shape (since the distinction between Outer and Common seems no longer relevant, after removing Inner)

Reply:
Updated in latest push.